packages/react-next/etc/react-next.api.md

```diff
@@ -4,27 +4,91 @@
 
 ```ts
 
+import { IBaseProps } from 'office-ui-fabric-react/lib/Utilities';
 import { IButtonProps } from 'office-ui-fabric-react/lib/Button';
 import { IButtonProps as IButtonProps_2 } from 'office-ui-fabric-react/lib/components/Button/Button.types';
 import { IButtonStyles } from 'office-ui-fabric-react/lib/Button';
+import { ICalloutContentStyleProps } from 'office-ui-fabric-react/lib/Callout';
+import { ICalloutProps } from 'office-ui-fabric-react/lib/Callout';
 import { IComponentAs } from 'office-ui-fabric-react/lib/Utilities';
-import { IFocusZoneProps } from '@fluentui/react-focus';
+import { IFocusZoneProps } from 'office-ui-fabric-react/lib/FocusZone';
+import { IFocusZoneProps as IFocusZoneProps_2 } from '@fluentui/react-focus';

```

Comment:
Can you fix this to avoid exporting things twice? (same with IIconProps)

Reply:
I brought up the underlying issue in [this comment](https://github.com/microsoft/fluentui/pull/12928#issuecomment-623692136)

> @dzearing @ecraig12345 Not sure of the advice concerning [this build error](https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=71364&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=cb33ee24-f492-54d2-727a-ae0a7bb84b2c&l=1096):
>```
>@fluentui/react-next: [8:02:30 PM] x src/index.ts:18:1 - error TS2308: Module './Calendar' >has already exported a member named 'DateRangeType'. Consider explicitly re-exporting to >resolve the ambiguity.
>@fluentui/react-next: 18 export * from './DatePicker';
>```
>
>Basically, since certain types are being re-exported from multiple components, and those types are now technically different when they're from different source files (OUFR vs react-next), we get this error.
>
>Should we just copy OUFR over to react-next wholesale?

My question remains. I can't simply fix the duplication without either
- copying over a substantial portion (for this error) or all (for all similar errors) of OUFR over to react-next, or
- breaking existing export chains.

Reply:
Similar to what I responded on the other PR, I think the right thing to do for now is manually fixing up any duplicates (modifying import paths and removing duplicate exports as needed).